xline/src/server/maintenance.rs

```diff
@@ -67,9 +81,32 @@ where
         &self,
         _request: tonic::Request<StatusRequest>,
     ) -> Result<tonic::Response<StatusResponse>, tonic::Status> {
-        Err(tonic::Status::unimplemented(
-            "status is unimplemented".to_owned(),
-        ))
+        let cluster = self
+            .client
+            .get_cluster_from_curp(false)
+            .await
+            .map_err(propose_err_to_status)?;
```

Comment:
Status should be get directly from the local server, not through client.

Reply:
Yes, the `false` parameter in `get_cluster_from_curp` will trigger to fetch status from local server first, if failed, then fetch the leader status.

Reply:
see above